hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HederaOperations.java

```diff
@@ -38,6 +38,9 @@
  * Provides the Hedera operations that only a {@link ProxyWorldUpdater} needs (but not a {@link DispatchingEvmFrameState}.
  */
 public interface HederaOperations {
+    /**
+     * Default contract id to return when the searched one is not found,
```

Comment:
this comment seems incorrect compared to how this constant is actually used.

Reply:
Does something like "A contract id to indicate that a given ContractId has a mismatch with the config" feels more appropriate? @lukelee-sl

Reply:
Changed to _A contract id to indicate that a given ContractId has a mismatch with the config_